share/com.github.baresip.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type =          Application
+Name =          baresip
+GenericName =   Telephony client
+Comment =       Baresip is a portable and modular SIP User-Agent with audio and video support.
+Categories =    Office;Telephony;
+
+Terminal =      true
+Exec =          baresip
+StartupNotify = true
```

Comment:
As of writing, I'm shipping this [`com.github.baresip.desktop`](https://src.fedoraproject.org/rpms/baresip/blob/rawhide/f/com.github.baresip.desktop) file in Fedora and EPEL, however optimized for X11 environments. I didn't investigate yet, whether a combined `*.desktop` is possible or whether downstreams need a wrapper in between (or even two different `*.desktop` files).

Reply:
In Wayland this file is now needed in one of the application folders for gnotification to show a notification.

Using `env GDK_BACKEND=x11 ` in the exec seems interesting. I will test how it changes the behavior.